markdown/blockprocessors.py

```diff
@@ -175,7 +175,7 @@ def test(self, parent: etree.Element, block: str) -> bool:
         return block.startswith(' '*self.tab_length) and \
             not self.parser.state.isstate('detabbed') and \
             (parent.tag in self.ITEM_TYPES or
-                (len(parent) and parent[-1] is not None and
+                (len(parent) > 0 and parent[-1] is not None and
```

Comment:
The code can **actually** sometimes return an instance of `int` rather than `bool`. If we have to, I'll follow the usual over-the-top process of proving to you that this is a bug and writing a test.